packages/form-fields/checkbox-group/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useId } from '@reach/auto-id'
+import Checkbox from '../checkbox'
+import Label from '../partials/label'
+import type { ComponentProps } from 'react'
+import classNames from 'classnames'
+import s from './style.module.css'
+
+interface CheckboxGroupProps {
+  label?: string
+  helpText?: string
+  required?: boolean
+  error?: string
+  appearance?: 'light' | 'dark'
+  className?: string
+  inputs: ComponentProps<typeof Checkbox>[]
+}
+
+function CheckboxGroup({
+  label,
+  error,
+  required,
+  helpText,
+  appearance,
+  className,
+  inputs,
+}: CheckboxGroupProps) {
+  const labelId = useId()
+  const helpId = useId()
+  return (
+    <div className={classNames(s.root, className, s[`theme-${appearance}`])}>
```

Comment:
Should we be using a fieldset and legend here?

Reply:
Yeah, semantically they're the right way to go here, but they're [difficult to style](https://thatemil.com/blog/2015/01/03/reset-your-fieldset/), so I went with an aria-based solution instead. This is a weak opinion loosely held, so I'm open to being swayed!

Reply:
Yeah, I think it's become easier to style these days. Margin, padding, and border reset on the fieldset should be sufficient. Seeing the same usage on polaris choicelist implementation https://polaris.shopify.com/components/choice-list

Reply:
This change was so easy I'm a bit embarrassed I didn't do it to begin with 😅 [`3c2840a` (#801)](https://github.com/hashicorp/react-components/pull/801/commits/3c2840a74d14cc71fe910bfed4b833e2716dbe67)